scheduler/flags/flags.go

```diff
@@ -206,6 +206,7 @@ var (
 		Value:  1,
 		EnvVar: prefixEnvVar("BLOCK_OFFSET"),
 	}
+	// is this the scheduler private key? why in optional flags?
```

Comment:
As far as I can tell, a Scheduler private key does not exist as a flag - but the Scheduler in the `consensus/clique` package has a Wallet field